primitives/src/v1.rs

```diff
@@ -119,6 +119,22 @@ pub mod well_known_keys {
 		})
 	}
 
+	/// The list of inbound channels for the given para.
+	///
+	/// The storage entry stores a `Vec<ParaId>`
+	pub fn hrmp_ingress_channel_index(para_id: Id) -> Vec<u8> {
+		let prefix = hex!["6a0da05ca59913bc38a8630590f2627c1d3719f5b0b12c7105c073c507445948"];
```

Comment:
what is this?

Reply:
That's a prefix. It's generated by the `decl_storage` for [this](https://github.com/paritytech/polkadot/blob/7aa88e366239edbca7dcb4ff73527a3efdde0fab/runtime/parachains/src/hrmp.rs%23L285) storage item. This is essentially the value you get when calling `StorageMap::prefix_hash`. Since we are so upstream here this is copied verbatim. 

Perhaps, a more friendlier version would be to generate this from the components, i.e. the module name and the field name.

Reply:
Gotcha - I think a comment explaining that would be helpful!

Reply:
There actually is [one](https://github.com/paritytech/polkadot/blob/deea097a3e3f4580f1195a3b2ac66c1ca422bcbb/primitives/src/v1.rs#L69-L79), but above. Do you think we should comment on each `prefix`?

Maybe it's possible to extract a function and slap a comment there.

Reply:
Oh, that's fine. It was just omitted from the diff